catalogue/webapp/test/e2e/images.test.ts

```diff
@@ -0,0 +1,25 @@
+import { imagesUrl } from './urls';
+import { fillSearchInput, pressEnterSearchInput } from './selectors/search';
+import { isMobile } from './selectors/common';
+import { toggleFeature } from './utils';
+
+describe('images', () => {
+  describe('New search', () => {
+    beforeEach(async () => {
+      await toggleFeature('searchPrototype', 'true');
```

Comment:
I thought that this is on by default. We should probably remove the toggle, and this part of the test.

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/pull/5895